lib/hexpm/web/views/dashboard_view.ex

```diff
@@ -35,6 +35,19 @@ defmodule Hexpm.Web.DashboardView do
     User.email(user, :public) || "none"
   end
 
+  def gravatar_email_options(user) do
+    emails =
+      user.emails
+      |> Enum.filter(&(&1.verified))
+      |> Enum.map(&{&1.email, &1.email})
+
+    [{"Don't use gravatar", "none"}] ++ emails
```

Comment:
Maybe "Don't show an avatar"?

Reply:
Maybe. Seems like better description of empty option.